app/commands/v2/put_content.rb

```diff
@@ -304,8 +307,13 @@ def raise_if_links_is_provided
         )
       end
 
-      def validate_schema
-        SchemaValidator.new(type: :schema).validate(payload.except(:content_id))
+      def schema_validation_errors
+        @_validation_errors ||=
```

Comment:
Is the underscore because you get a linting error otherwise or just to provide a clue that this ivar isn't used elsewhere? just flagging it as we've not got any other `@_` vars in the codebase